config/services.yaml

```diff
@@ -5,6 +5,7 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
   locale: 'en'
+  user_image_directory: '%kernel.project_dir%/src/Images/User/'
```

Comment:
This should be an in the `public` folder, not `src`. `src` is not available publicly.